**_Summary: Whole-Genome Sequencing Accurately Identifies_**

**_Resistance to Extended-Spectrum β-Lactams for Major Gram-Negative Bacterial Pathogens_**

**_ _**

**Authors (@Slack):**

** **

**Ayesha Noor ( @AyeshaNoor), Cateline Atieno Ouma (@Cateline), Mahesh Rani Kamilus (), Nada Esmael Sleim (@Eddy27)**

** **

**Video Presentation:**

**https\://www\.linkedin.com/posts/cateline-ouma-6470aa1b2\_as-part-of-the-hackbio-cancer-internship-activity-7238264755675664384-LY1Y?utm\_source=share\&utm\_medium=member\_android**

**Introduction**

In the realm of modern microbiology, the ability to swiftly and accurately identify antimicrobial resistance (AMR) is paramount, particularly in the face of rising infections caused by resistant pathogens. As we stand at the crossroads of traditional and cutting-edge technologies, the integration of Whole-Genome Sequencing (WGS) emerges as a transformative approach in the fight against AMR. This study delves into the application of WGS to predict AMR in gram-negative bacteria isolated from patients with neutropenic fever—a group particularly vulnerable to severe infections. By leveraging WGS, this research not only addresses the limitations of conventional phenotypic testing methods but also pioneers a new frontier in microbial genomics. Through a comprehensive analysis of AMR determinants and their genetic underpinnings, we aim to redefine the paradigms of bacterial resistance prediction, offering unprecedented accuracy and insight into the mechanisms that drive resistance. This work stands as a testament to the power of genomic technology to reshape our understanding and management of antimicrobial resistance, setting the stage for a future where precision medicine becomes the norm in combating infectious diseases.

 

**Objective and Methodology**

This study focuses on evaluating the efficacy of whole-genome sequencing (WGS) for predicting antimicrobial resistance (AMR) in gram-negative bacteria, using neutropenic fever as a model system. The research aimed to address the increasing need for rapid and accurate AMR detection, a challenge compounded by the rise of antimicrobial-resistant infections. The study involved strains of _E. coli_, _Klebsiella pneumoniae_, _Pseudomonas aeruginosa_, and _Enterobacter cloacae_, selected for their varied resistance profiles. WGS was performed on these strains to identify AMR mechanisms, and the results were compared to traditional susceptibility testing methods, including broth microdilution (BMD) and commercial microbiology techniques.

 

 

**Whole-Genome Sequencing and Analysis**

Strains were chosen based on their broad range of antimicrobial resistance phenotypes. A custom database of AMR protein sequences was constructed by merging data from the Antibiotic Resistance Database (ARDB) and Comprehensive Antibiotic Resistance Database (CARD), focusing on β-lactamase genes and their mutations. WGS identified 133 instances of predicted AMR out of 360 combinations of strains and β-lactams. These were classified into two main categories: exogenous β-lactamases and chromosomal gene mutations. The study found that traditional PCR methods could detect only 65% of these mechanisms, underscoring the comprehensive nature of WGS.

 

**Phenotypic Analysis**

Phenotypic susceptibility testing revealed a range of minimum inhibitory concentrations (MICs) for β-lactams across species, with notable variability and some instances of resistance. The data showed no resistance to meropenem in _E. cloacae_, and high resistance rates in _P. aeruginosa_ and _K. pneumoniae_. These findings demonstrated the heterogeneity of the bacterial strains and validated the utility of WGS in assessing AMR.

 

**Comparison with Traditional Methods**

WGS predictions showed high concordance with BMD results (93% agreement), outperforming traditional commercial methods. Specifically, WGS demonstrated higher positive predictive value (0.97 vs. 0.92) and similar sensitivity compared to commercial methods. The study highlighted the variability in MICs reported by commercial methods, particularly for _E. coli_ and _K. pneumoniae_, where reference methods often identified higher MICs for strains carrying CTX-M enzymes.

**Conclusion**

Overall, the study provides strong evidence supporting the use of WGS for AMR prediction in clinical settings. It demonstrates that WGS offers a more comprehensive and accurate assessment of AMR compared to traditional susceptibility testing methods. As WGS technology advances, it has the potential to become a standard tool in clinical microbiology, improving the rapid identification of AMR and guiding effective treatment strategies. Future work should focus on expanding the range of antibiotics studied, automating data analysis processes, and validating WGS across different healthcare environments.

**_ _**

**_Reference_**

<!--[if !supportLists]-->1-    <!--[endif]-->Shelburne SA, Kim J, Munita JM, Sahasrabhojane P, Shields RK, Press EG, Li X, Arias CA, Cantarel B, Jiang Y, Kim MS, Aitken SL, Greenberg DE. Whole-Genome Sequencing Accurately Identifies Resistance to Extended-Spectrum β-Lactams for Major Gram-Negative Bacterial Pathogens. Clin Infect Dis. 2017 Sep 1;65(5):738-745. doi: 10.1093/cid/cix417. PMID: 28472260; PMCID: PMC5850535.

** **
